src/dune/foreign.mli

```diff
@@ -70,6 +70,49 @@ val possible_sources :
   -> dune_version:Dune_lang.Syntax.Version.t
   -> string list
 
+module Archive : sig
+  module Name : sig
```

Comment:
Let's leave a comment to describe the purpose of this module.

Reply:
Done in the upcoming commit.